homeassistant/components/kmtronic/__init__.py

```diff
@@ -90,6 +106,14 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
         )
     )
     if unload_ok:
+        update_listener = hass.data[DOMAIN][entry.entry_id][UPDATE_LISTENER]
+        update_listener()
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+def _get_config_value(config_entry, key, default):
```

Comment:
This doesn't seem needed. We can just do `entry.options.get(CONF_REVERSE, False)`, where we need to get access to that option.

Reply:
I made this in order to fallback to the config entry configuration, but I now see that I dropped CONF_REVERSE from the configflow. Will remove